source/features/delete-review-comments-faster.tsx

```diff
@@ -0,0 +1,46 @@
+import './delete-review-comments-faster.css';
+
+import React from 'dom-chef';
+import select from 'select-dom';
+import onetime from 'onetime';
+import oneEvent from 'one-event';
+import delegate from 'delegate-it';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function onButtonClick(event: delegate.Event): Promise<void> {
+	const comment = event.delegateTarget.closest<HTMLElement>('.js-comment')!;
+	const dropdownFragment = select('include-fragment.SelectMenu-loading', comment);
+	if (dropdownFragment) {
+		select('.timeline-comment-actions > details:last-of-type', comment)!.dispatchEvent(new Event('mouseover'));
+		await oneEvent(dropdownFragment, 'load');
+	}
+
+	select('.dropdown-menu .js-comment-delete > button', comment)!.click();
+}
+
+function init(): void {
+	delegate(document, '.rgh-review-comment-delete-button', 'click', onButtonClick);
+	observe('.review-comment > .unminimized-comment form:not(.js-single-suggested-change-form) .btn-primary[type="submit"]:not(.rgh-delete-button-added)', {
+		add(submitButton) {
```

Comment:
We should probably start moving these functions to the top level instead of keeping them inline here